elftools/elf/enums.py

```diff
@@ -573,9 +582,13 @@
     DT_MIPS_HIPAGENO=0x70000014,
     DT_MIPS_RLD_MAP=0x70000016,
     DT_MIPS_RLD_MAP_REL=0x70000035,
-    DT_AUXILIARY=0x7ffffffd,
-    DT_FILTER=0x7fffffff,
-    _default_=Pass,
+)
+
+ENUMMAP_EXTRA_D_TAG_MACHINE = dict(
+    EM_MIPS=ENUM_D_TAG_MIPS,
+    EM_MIPS_RS3_LE=ENUM_D_TAG_MIPS,
+    # TODO: add the rest
+    # solaris doesn't go here because what matters is e_ident[EI_OSABI] == ELFOSABI_SOLARIS
```

Comment:
I'm not sure I follow this comment. What are you trying to say? It may be obvious that we now have a separate section for the solaris tags, so maybe it's superfluous?